hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -347,6 +347,38 @@ public AbfsConfiguration(final Configuration rawConfig, String accountName)
     }
   }
 
+  @BooleanConfigurationValidatorAnnotation(
+          ConfigurationKey = FS_AZURE_REDIRECT_RENAME,
+          DefaultValue = DEFAULT_FS_AZURE_REDIRECT_RENAME)
+  private boolean redirectRename;
+
+  @BooleanConfigurationValidatorAnnotation(
+          ConfigurationKey = FS_AZURE_REDIRECT_DELETE,
+          DefaultValue = DEFAULT_FS_AZURE_REDIRECT_DELETE)
+  private boolean redirectDelete;
+
+  public boolean shouldRedirectRename() {
+    return this.redirectRename;
+  }
+
+  public boolean shouldRedirectDelete() {
+    return this.redirectDelete;
+  }
+
+  public boolean isRedirection() {
+    return (shouldRedirectRename() || shouldRedirectDelete());
+  }
+
+  @VisibleForTesting
+  public void setRedirectRename(boolean redirectRename) {
+    this.redirectRename = redirectRename;
+  }
+
+  @VisibleForTesting
+  public void setRedirectDelete(boolean redirectDelete) {
+    this.redirectDelete = redirectDelete;
+  }
```

Comment:
not used. lets remove it; atleast not have it public.

Reply:
Done